### ISSUE TITLE
Stats: show top contributors for browse pages under TP level properly 

### ISSUE THREAD

**Issue by ta2-1** [translate/pootle#4756]
This PR #4696 exposed leaders board at all browse pages but we display TP stats for any page under the TP.
Let's discuss here if it's necessary to filter top contributors stats by pootle_path under TP.

**Comment by dwaynebailey** [comment]
I agree, for the simple reason that the expected behaviour should be consistent.
